tkupchyn/homework_05/non_unique_elements.py

```diff
@@ -0,0 +1,15 @@
+from collections import Counter
+from typing import Iterable
+
+
+def remove_unique_values(iterable: list) -> Iterable:
+    """
+    Removes unique values from a list and returns an iterable with only non-unique values.
+
+    Args:
+        iterable (list): a non-empty sequence of integers.
+    Returns:
+        Iterable: Iterable consisting of only the non-unique elements from the initial sequence
+    """
+    counts = Counter(iterable)
+    return (elem for elem in iterable if counts[elem] > 1)
```

Comment:
iterate by keys in counter:


(k for k in counter.items() if v > 1)

Reply:
solve also without counter

Reply:
Added alternative solution without counter

> iterate by keys in counter:
> 
> (k for k in counter.items() if v > 1)

This part will not work in our case, because we need to return the the non-unique elements in the same order. Example: [1, 3, 1, 3] . If we iterate through the counter.items() we'll get  only [1, 3]

Reply:
oh you need just to filter existing elements by count, got it

Reply:
counter looks like overkill for this task )